lib/HTTP/Tiny.pm

```diff
@@ -40,8 +40,10 @@ This constructor returns a new HTTP::Tiny object.  Valid attributes include:
 * C<timeout> — Request timeout in seconds (default is 60) If a socket open,
   read or write takes longer than the timeout, the request response status code
   will be 599.
-* C<verify_SSL> — A boolean that indicates whether to validate the SSL
-  certificate of an C<https> — connection (default is false)
+* C<verify_SSL> — A boolean that indicates whether to validate the TLS/SSL
+  certificate of an C<https> — connection (default is true)
```

Comment:
This and the environment variable below should note the version of HTTP::Tiny at which this default changed.  People might read current docs online but have older versions installed.